src/components/Organisms/TodoCard/index.jsx

```diff
@@ -34,6 +34,13 @@ const TodoCard = () => {
       return null;
     }
   });
+  useEffect(() => {
+    if (JSON.parse(window.localStorage.getItem("task")) === null) return;
+    setNewTask(JSON.parse(window.localStorage.getItem("task")));
```

Comment:
```suggestion
    const task = JSON.parse(window.localStorage.getItem("task")) 
    if (task === null) {
      return
    };
    setNewTask(task);
```
2回以上同じ処理を行なっている場合は変数に格納した方がいいでしょう。
また、if分の中括弧を省略する書き方は可読性の観点から避けるべきという意見もあるため、早期リターンを行う場合も中括弧を展開するのが丸いのかなと思っています。